### PR TITLE
Add Fizzbuzz in Forth and COBOL

### PR DESCRIPTION
**Pull request by AngrySoundTech**

### CODE REVIEW

**Comment by nloveladyallen** [comment]
COBOL version outputs:

    001
    002
    Fizz
    004
    Buzz
    ...

Is there a way to eliminate the leading 0s?

**Comment by nloveladyallen** [comment]
Also, Forth version should use `/usr/bin/env gforth` instead of `/usr/bin/gforth`

File: forth.fth [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env gforth
```
Comment (nloveladyallen) [inline forth.fth]: There needs to be a space between the `#!` and the `/` or the Forth interpreter won't accept it

Comment (AngrySoundTech) [inline forth.fth]: Huh, I tested it on Arch Linux 2017.02.01 and this shebang worked, which it probably shouldn't have.

**Comment by AngrySoundTech** [comment]
This became quite a few small commits/fixes, Would you like for them to be squashed into one?

**Comment by nloveladyallen** [comment]
Sorry for the delay.

The space-padding solution produces output that looks right to the eye, but unfortunately it doesn't look right to `./cobol | diff - text.txt`, which is how I generally test all PRs.

Of course, we could just make an exception to the rule that programs should output the contents of text.txt exactly—there's no hard and fast rules (really no rules at all) for what can or cannot be included here, and it seems reasonable to make an exception for a language as antiquated as COBOL. 

I'll defer to @zenware for thoughts on this.

**Comment by AngrySoundTech** [comment]
The issue with COBOL in this case is that the result string in working storage has to be preformatted to take up three characters. After a little bit of looking around, most commonly accepted solutions will have their output as` 001, 002, Fizz, etc.`

**Comment by zenware** [comment]
Interesting, I'll have a look at this later... I know a bit of COBOL

**Pull request merged**